homeassistant/components/intellifire/strings.json

```diff
@@ -1,12 +1,16 @@
 {
   "config": {
+    "flow_title": "Discovered Fireplace ({host})",
```

Comment:
The flow will already have the integration name and that it's discovered so we should drop discovered fireplace.  Maybe show them {serial} as well?

Reply:
Will-do!